### PR TITLE
Minor formatting in the spreadsheet subchapter

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* This PR fixes the formatting error in the RDM file recently written by @jcolomb 

### List of changes proposed in this PR (pull-request)

* This has numbering of the points described which did not appear correctly in the original file

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@Ismael-KG would you be able to review this PR?

**Comment by jcolomb** [comment]
I would put the extra tips after the ## 4.

File: book/website/reproducible-research/rdm/rdm-spreadsheets.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Finally, you should also be aware of the default behaviour of your spreadsheet p
 For instance, the decimal is usually indicated with a comma in the French or German version of excel, but while a dot in English versions where the comma has no meaning
 (`9,000`  will be translated into 9000 or 9 depending on the version you are using).
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm/rdm-spreadsheets.md:91]:
```suggestion
## 4. Data manipulation and analysis

***Do not manipulate or analyse data in a spreadsheet program.***

In particular, only copy-paste from one spreadsheet to another if the process is used very rarely. 
It is now very easy to read and combine the different spreadsheets in the analysis software, with the additional advantage that the software will return an error message if the headers do not fit.
```

Comment (jcolomb) [reply book/website/reproducible-research/rdm/rdm-spreadsheets.md:91]: can't find how to delete the part 4 below

Comment (malvikasharan) [reply book/website/reproducible-research/rdm/rdm-spreadsheets.md:91]: You should be able to directly edit the file here: https://github.com/alan-turing-institute/the-turing-way/blob/malvikasharan-rdm-formatting/book/website/reproducible-research/rdm/rdm-spreadsheets.md

Comment (jcolomb) [reply book/website/reproducible-research/rdm/rdm-spreadsheets.md]: this would make a new PR...

Comment (malvikasharan) [reply book/website/reproducible-research/rdm/rdm-spreadsheets.md]: It shouldn't. You will be able to directly edit in the branch that will appear in this PR.

Comment (jcolomb) [reply book/website/reproducible-research/rdm/rdm-spreadsheets.md]: ... made a new PR.

Comment (malvikasharan) [reply book/website/reproducible-research/rdm/rdm-spreadsheets.md:91]: merged to this 🌮

**Comment by malvikasharan** [comment]
@jcolomb should we ask for approval to merge this PR?

**Review by sgibson91 — APPROVED** [review]
🤖

**Pull request merged**